contrib/pg_tde/documentation/docs/functions.md

```diff
@@ -201,12 +201,12 @@ Use these functions to create a new principal key for a specific scope such as a
 
 Princial keys are stored on key providers by the name specified in this function - for example, when using the Vault provider, after creating a key named "foo", a key named "foo" will be visible on the Vault server at the specified mount point.
 
-### pg_tde_set_principal_key_using_database_key_provider
+### pg_tde_set_key_using_database_key_provider
 
 Creates or rotates the principal key for the current database using the specified database key provider and key name.
```

Comment:
Do we keep the notion of a principal key?

Reply:
Probably yes. They still `principal` by meaning, but they are only available to the user.

What do you think @dutow ?

Reply:
yes, it is still called principal key, we just simplified the names of the functions.

Reply:
Sounds good to me too. It makes things more clear to use the long form "principal key" in the documentation.